configure.ac

```diff
@@ -886,12 +886,41 @@ AS_IF([test "x$enable_ecap" != "xno"],[
   ])
 
   SQUID_STATE_SAVE(squid_ecap_state)
+  CPPFLAGS="$EXT_LIBECAP_CFLAGS $CPPFLAGS"
+  LIBS="$EXT_LIBECAP_LIBS $LIBS"
+  AC_CHECK_HEADERS([ \
+    libecap/adapter/service.h \
+    libecap/adapter/xaction.h \
+    libecap/common/area.h \
+    libecap/common/body.h \
+    libecap/common/delay.h \
+    libecap/common/forward.h \
+    libecap/common/header.h \
+    libecap/common/memory.h \
+    libecap/common/message.h \
+    libecap/common/name.h \
+    libecap/common/named_values.h \
+    libecap/common/names.h \
+    libecap/common/options.h \
+    libecap/common/registry.h \
+    libecap/common/version.h \
+    libecap/host/host.h \
+    libecap/host/xaction.h \
+  ],,,[
+/* libecap-1.0.1 headers do not build without autoconf.h magic */
+#define HAVE_CONFIG_H
+/* libecap/common/delay.h fails to include <string> */
```

Comment:
libecap apparently lacks basic self-consistency checks for its installed headers.
AC_CHECK_HEADERS exposes a number of identical `std::string` errors, one example:
```
/usr/include/libecap/common/delay.h:15:43: error: 'string' in namespace 'std' does not name a type
   15 |                 explicit Delay(const std::string &state_):
      |                                           ^~~~~~
/usr/include/libecap/common/delay.h:7:1: note: 'std::string' is defined in header '<string>'; this is probably fixable by adding '#include <string>'
```